README.md

```diff
@@ -115,6 +116,58 @@ proxy.
   script. Hit the Connect button, enter a password if the VNC server has one
   configured, and enjoy!
 
+### Installation from Snap Package
+To install from Snap:
+
+`sudo snap install novnc`
+
+#### Running noVNC
+
+You can run the Snap-package installed novnc directly with, for example:
+`novnc --listen 6081 --vnc localhost:5901 # /snap/bin/novnc if /snap/bin is not in your PATH`
+
+#### Running as a Service (Daemon)
+The Snap package also has the capability to run a 'novnc' service which can be configured to listen on multiple ports connecting to multiple VNC servers (effectively a service runing multiple instances of novnc). Instructions (with example values):
```

Comment:
We've tried to keep the lines over 80 columns to keep it sane in consoles. Could you break your lines to that as well?

Exceptions are allowed when breaking them isn't really feasible, e.g. an example command line.

Reply:
Should be fixed in latest commit